### PR TITLE
fix(UI): change-update Look and Feel

### PR DESCRIPTION
**Pull request by katiamiyu**
### What does this PR do?
Change the links on the landing page
Update the menus on the Service page to open the required pages
#### Description of Task to be completed?
1. Update the navigation links of landing page
2. Update menus of service page to link their respective pages
3. Adjust the username input box on the signup page
4. Adjust the grid view on the account and users page
#### How should this be manually tested?
Clone the repository
open the bank folder and double click index.html
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2324355
Story id: #165372481
Story: Feedback - Look and feel on homepage

[Fixes #165372481]

### CODE REVIEW

File: ui/assets/css/accounts.css [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+}
+ 
+ /* gridview */
+ .row{
+    overflow: hidden;
+    box-sizing: content-box;
+    width:100%;
+    background-color: #fff;
+}
+/* .row{
```
Comment (ajimae) [inline ui/assets/css/accounts.css]: Please delete this commented code if you don't need it.

File: ui/assets/css/accounts.css [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+}
+@media screen and (max-width:760px){
+     /* gridview */
+     .row{
+        overflow: hidden;
+        box-sizing: content-box;
+        width:100%;
+        background-color: #fff;
+    }
+    /* .row{
```
Comment (ajimae) [inline ui/assets/css/accounts.css]: Delete this also.

File: ui/assets/css/style.css [hunk 3]
```diff
@@ -57,11 +55,17 @@ nav ul li{
...
 }
 nav ul li a{
-    text-decoration: none;
+    font-weight: bold;
+    border: 1px solid #fff;
+    padding: 5px 20px;
+    color: #111;
+    margin: 2px;
+    transition: 0.6s ease;
+    /* text-decoration: none;
```
Comment (ajimae) [inline ui/assets/css/style.css]: Please delete this comment lines of code also.

File: ui/assets/css/usergrid.css [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+    }
+@media screen and (max-width:760px){
+     /* gridview */
+     .row{
+        overflow: hidden;
+        box-sizing: content-box;
+        width:100%;
+        background-color: #fff;
+    }
+    /* .row{
```
Comment (ajimae) [inline ui/assets/css/usergrid.css]: Remove these commented code too

**Review by ajimae — CHANGES_REQUESTED** [review]
Nice work @katiamiyu please take a look at your PR, I left some comments.

File: ui/assets/css/accounts.css (hunk 2 above)
Comment (katiamiyu) [reply ui/assets/css/accounts.css]: Deleted

File: ui/assets/css/style.css (hunk 3 above)
Comment (katiamiyu) [reply ui/assets/css/style.css]: Deleted

**Pull request merged**